pkg/controller/credentialsrequest/status_test.go

```diff
@@ -182,20 +205,58 @@ func TestClusterOperatorVersion(t *testing.T) {
 
 			if err := fakeClient.Get(context.TODO(), client.ObjectKey{Name: cloudCredClusterOperator}, clusterop); err != nil {
 				t.Errorf("error fetching clusteroperator object: %v", err)
-			} else {
-				foundVersion := false
-				for _, version := range clusterop.Status.Versions {
-					if version.Name == "operator" {
-						foundVersion = true
-						assert.Equal(t, test.releaseVersionEnv, version.Version)
-					}
+				return
+			}
+
+			foundVersion := false
+			for _, version := range clusterop.Status.Versions {
+				if version.Name == "operator" {
+					foundVersion = true
+					assert.Equal(t, test.releaseVersionEnv, version.Version)
 				}
-				assert.True(t, foundVersion, "didn't find an entry named 'operator' in the version list")
+			}
+			assert.True(t, foundVersion, "didn't find an entry named 'operator' in the version list")
+
+			progCond := findClusterOperatorCondition(clusterop.Status.Conditions,
+				configv1.OperatorProgressing)
+			assert.NotNil(t, progCond)
+			if test.expectProgressingTransition {
+				assert.True(t, progCond.LastTransitionTime.Time.After(
+					test.currentProgressingLastTransition.Time))
+			} else {
+				assert.Equal(t, progCond.LastTransitionTime.Time.Format(time.UnixDate),
```

Comment:
assert.Equal() wants the expected value to be the second param, and the tested value to be the third param.